services/121-service/src/registration/helpers/registration-attribute-data-query.helper.spec.ts

```diff
@@ -0,0 +1,156 @@
+import { HttpException } from '@nestjs/common';
+import { FindOperatorType } from 'typeorm';
+
+import { RegistrationFilterQueryHelpers } from '@121-service/src/registration/helpers/registration-attribute-data-query.helper';
+import { RegistrationViewEntity } from '@121-service/src/registration/registration-view.entity';
+import { ScopedQueryBuilder } from '@121-service/src/scoped.repository';
+
+describe('RegistrationFilterQueryHelpers', () => {
+  let andWhereMock: jest.Mock;
+  let queryBuilder: ScopedQueryBuilder<RegistrationViewEntity>;
+
+  beforeEach(() => {
```

Comment:
Not super sure how valuable this is next to the api tests we already have.. What do you think @reviewer

Reply:
I think this is a nice clean and simple overview and assertion. Let's leave it in.